include/nil/blueprint/blueprint/plonk/assignment_proxy.hpp

```diff
@@ -198,20 +211,17 @@ namespace nil {
             }
 
             value_type constant(std::uint32_t constant_index, std::uint32_t row_index) const override {
-                if (check && used_rows.find(row_index) == used_rows.end()) {
-                    std::cout << id << ": Not found constant " << constant_index << " on row " << row_index << std::endl;
-                    BLUEPRINT_ASSERT(false);
+                if (check) {
+                    const auto lookup_constant_cols = assignment_ptr->get_lookup_constant_cols();
```

Comment:
Again, unsure if this is supposed to only check the `lookup_constant_cols`: what would happen to constants used in components?